### PR TITLE
[IMPROVEMENT] Nuget Readme included in the package

### PR DESCRIPTION
**Pull request by moljac**
### Does this change any of the generated binding API's?

No.

### Describe your contribution

NuGet best practices recommend adding `readme.[txt,md]` to each package. Otherwise following warning is emitted:

```
The package  is missing a readme. Go to https://aka.ms/nuget/authoring-best-practices/readme to learn why package readmes are important.
```

This PR adds `readme.md` to NuGet packages.

https://devblogs.microsoft.com/nuget/add-a-readme-to-your-nuget-package/

https://devblogs.microsoft.com/nuget/write-a-high-quality-readme-for-nuget-packages/

https://github.com/NuGet/Home/issues/12070

https://raw.githubusercontent.com/lyndaidaii/NuGetReadMeTemplate/main/README.md

Verification
```
```
Result:
```
```

### CODE REVIEW

File: source/GooglePlayServicesNuGetReadMe.cshtml [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+## About
+
+<!-- 
+    A description of the package and where one can find more documentation 
+-->
+
+Google.Play.Services, Firebase and ML Kit (GPS-FB-MLKit) Bindings NuGet Package @(Model.NuGetPackageId) 
+
+.NET for Android (formerly Xamarin.Android) GPS-FB-MLKit bindings NuGet Package @(Model.NuGetPackageId)
```
Comment (jpobst) [inline source/GooglePlayServicesNuGetReadMe.cshtml:33]: The name they finally decided on is `.NET Android`.

File: source/GooglePlayServicesNuGetReadMe.cshtml [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+<!-- 
+    A compelling example on how to use this package with code 
+-->
+
+See samples:
+
+https://github.com/xamarin/GooglePlayServicesComponents/tree/main/samples
+
+## Key Features
```
Comment (jpobst) [inline source/GooglePlayServicesNuGetReadMe.cshtml:57]: Let's remove the headings for empty sections.

File: source/GooglePlayServicesNuGetReadMe.cshtml [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+https://www.nuget.org/packages?q=Xamarin.Firebase
+
+https://www.nuget.org/packages?q=Xamarin.MLKit
+
+## Feedback & Contributing
+
+<!-- 
+    How to provide feedback on this package and contribute to it 
+-->
+https://github.com/xamarin/AndroidX/issues
```
Comment (jpobst) [inline source/GooglePlayServicesNuGetReadMe.cshtml:147]: Probably don't need a link to the AndroidX issues.

File: source/GooglePlayServicesNuGetReadMe.cshtml [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+## How to Use
+
+<!-- 
+    A compelling example on how to use this package with code 
+-->
+
+See samples:
+
+https://github.com/xamarin/GooglePlayServicesComponents/tree/main/samples
```
Comment (jpobst) [inline source/GooglePlayServicesNuGetReadMe.cshtml:55]: Do we feel our samples are up-to-date and complete enough that we should be recommending them? Or should we just leave this section out?

Comment (moljac) [reply source/GooglePlayServicesNuGetReadMe.cshtml:55]: Well... No they are at least outdated... Samples need tons of love.

Comment (moljac) [reply source/GooglePlayServicesNuGetReadMe.cshtml:55]: removed

File: source/GooglePlayServicesNuGetReadMe.cshtml (hunk 3 above)
Comment (moljac) [reply source/GooglePlayServicesNuGetReadMe.cshtml:147]: true. Removed

File: source/GooglePlayServicesNuGetReadMe.cshtml (hunk 2 above)
Comment (moljac) [reply source/GooglePlayServicesNuGetReadMe.cshtml:57]: removed

**Review by jpobst — APPROVED** [review]

**Pull request merged**